main.go

```diff
@@ -21,9 +24,41 @@ var (
 	cachedServiceResults          []byte
 	cachedServiceResultsGzipped   []byte
 	cachedServiceResultsTimestamp time.Time
+	port                          int
+	host                          string
 )
 
+func init() {
+	// Customizing priority:
+	// (1) command line parameters will be preferred over
+	// (2) environment variables will be preferred over
+	// (3) application defaults
+
+	// set defaults for the case that neither an environment variable nor a
+	// command line parameter is passed
+	var defaultHost = ""
+	var defaultPort = 8080
+
+	// assume set if the is a valid port number
+	if p, err := strconv.Atoi(os.Getenv("GATUS_CONFIG_PORT")); err == nil && p > 0 {
+		defaultPort = p
+	}
+
+	// explicitly asked if the user has set a the environment variable to
+	// blank / empty in order to allow listening on all interfaces
+	if h, set := os.LookupEnv("GATUS_CONFIG_HOST"); set == true {
+		defaultHost = h
+	}
+
+	flag.IntVar(&port, "port", defaultPort, "port to listen (default: 8080)")
+	flag.IntVar(&port, "p", defaultPort, "port to listen (default: 8080 ; shorthand)")
+	flag.StringVar(&host, "host", defaultHost, "host to listen on (default all interfaces on host)")
+	flag.StringVar(&host, "h", defaultHost, "host to listen on (default all interfaces on host; shorthand)")
```

Comment:
For the sake of consistency, this kind of configuration should probably be in the configuration file instead.

e.g.
```yaml
port: 8080
host: 0.0.0.0
```

Reply:
This is where the `port` and `host` variables should be added: https://github.com/TwinProduction/gatus/blob/be72a73082e1e04f4552440eac0caef08bb38970/config/config.go#L44

A new function for validating the port (e.g. `validatePortConfig`) that would automatically set `config.Port` to `8080` if it's not set (`0`) should be added here: https://github.com/TwinProduction/gatus/blob/be72a73082e1e04f4552440eac0caef08bb38970/config/config.go#L129

As for the host, I don't think there's a need to add a validation for it since if it's not set, it will default to an empty string, which is a valid value.

Reply:
Try to implement the requested changes - hopefully it is more idiomatic now. Thanks for review and commenting.